### PR TITLE
Add my Fastify masterclass video to the talks list

### PR DESCRIPTION
**Pull request by mcollina**
PTAL

### CODE REVIEW

**Review by gurgunday — APPROVED** [review]
lgtm

File: src/pages/resources.mdx [hunk 1]
```diff
@@ -33,3 +33,5 @@ Watch these talks to learn more about Fastify:
 - [Monoliths always pay their debts - Tomas Della Vedova - NodeJS Italian Conference 2018](https://www.youtube.com/watch?v=6ucyzQULlWw)
 
 - [NodeConf Remote 2020 - Luca Maraschi - From mess to mesh using Fastify](https://www.youtube.com/watch?v=l3mQfvq07CI)
+
+- [Fastify Masterclass - Matteo Collina - 2024](https://www.youtube.com/watch?v=x3SG71Ut2tA&t=1s)
```
Comment (Fdawgs) [inline src/pages/resources.mdx]: Not a big deal but is the `t=1s` start time param needed?

Comment (Uzlopak) [reply src/pages/resources.mdx]: Well, it saves a second from the viewer. ;)

Comment (mcollina) [reply src/pages/resources.mdx]: fixed

**Review by Uzlopak — APPROVED** [review]
LGTM

**Review by Fdawgs — APPROVED** [review]

**Pull request merged**